nnotepad/css/index.css

```diff
@@ -0,0 +1,416 @@
+      format("svg"); /* Legacy iOS */
+}
+
+/* poppins-italic - latin */
+@font-face {
+  font-family: "Poppins";
+  font-style: italic;
+  font-weight: 400;
+  src: url("../common/css/fonts/poppins/poppins-v15-latin-italic.eot"); /* IE9 Compat Modes */
+  src: local(""),
+    url("../common/css/fonts/poppins/poppins-v15-latin-italic.eot?#iefix")
+      format("embedded-opentype"),
+    /* IE6-IE8 */
+      url("../common/css/fonts/poppins/poppins-v15-latin-italic.woff2")
+      format("woff2"),
+    /* Super Modern Browsers */
+      url("../common/css/fonts/poppins/poppins-v15-latin-italic.woff")
+      format("woff"),
+    /* Modern Browsers */
+      url("../common/css/fonts/poppins/poppins-v15-latin-italic.ttf")
+      format("truetype"),
+    /* Safari, Android, iOS */
+      url("../common/css/fonts/poppins/poppins-v15-latin-italic.svg#Poppins")
+      format("svg"); /* Legacy iOS */
+}
+
+/* poppins-500 - latin */
+@font-face {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  src: url("../common/css/fonts/poppins/poppins-v15-latin-500.eot"); /* IE9 Compat Modes */
+  src: local(""),
+    url("../common/css/fonts/poppins/poppins-v15-latin-500.eot?#iefix")
+      format("embedded-opentype"),
+    /* IE6-IE8 */ url("../common/css/fonts/poppins/poppins-v15-latin-500.woff2")
+      format("woff2"),
+    /* Super Modern Browsers */
+      url("../common/css/fonts/poppins/poppins-v15-latin-500.woff")
+      format("woff"),
+    /* Modern Browsers */
+      url("../common/css/fonts/poppins/poppins-v15-latin-500.ttf")
+      format("truetype"),
+    /* Safari, Android, iOS */
+      url("../common/css/fonts/poppins/poppins-v15-latin-500.svg#Poppins")
+      format("svg"); /* Legacy iOS */
+}
+
+/* poppins-500italic - latin */
+@font-face {
+  font-family: "Poppins";
+  font-style: italic;
+  font-weight: 500;
+  src: url("../common/css/fonts/poppins/poppins-v15-latin-500italic.eot"); /* IE9 Compat Modes */
+  src: local(""),
+    url("../common/css/fonts/poppins/poppins-v15-latin-500italic.eot?#iefix")
+      format("embedded-opentype"),
+    /* IE6-IE8 */
+      url("../common/css/fonts/poppins/poppins-v15-latin-500italic.woff2")
+      format("woff2"),
+    /* Super Modern Browsers */
+      url("../common/css/fonts/poppins/poppins-v15-latin-500italic.woff")
+      format("woff"),
+    /* Modern Browsers */
+      url("../common/css/fonts/poppins/poppins-v15-latin-500italic.ttf")
+      format("truetype"),
+    /* Safari, Android, iOS */
+      url("../common/css/fonts/poppins/poppins-v15-latin-500italic.svg#Poppins")
+      format("svg"); /* Legacy iOS */
+}
+
+/* victor-mono-regular - latin */
+@font-face {
+  font-display: swap; /* Check https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display for other options. */
+  font-family: "Victor Mono";
+  font-style: normal;
+  font-weight: 400;
+  src: url("../common/css/fonts/victor-mono-v3-latin-regular.woff2")
```

Comment:
`"../common/css/fonts/victor-mono-v3-latin-regular.woff2"` 

should be

`"../../common/css/fonts/victor-mono-v3-latin/victor-mono-v3-latin-regular.woff2"`

(note extra directory)